eng/Signing.props

```diff
@@ -6,5 +6,9 @@
     <FileExtensionSignInfo Include=".otf" CertificateName="None" />
     <FileExtensionSignInfo Include=".ttf" CertificateName="None" />
     <FileExtensionSignInfo Include=".msu" CertificateName="None" />
+    <!-- Ignore these files. These are signed or have tampered signatures for specific test cases and should NOT be signed again. -->
+    <ItemsToSign Remove="triple_signed.dll" />
+    <ItemsToSign Remove="dual_signed.dll" />
+    <ItemsToSign Remove="tampered.msi" />

```

Comment:
I don't think you can exclude them this way, since it is the nupkg files that are in `ItemsToSign`, not the contents of each package (the signing task unpacks / signs contents / repacks). It looks like all the contents in `Microsoft.DotNet.Installer.Windows.Security.TestData` should be excluded, so you can probably exclude the entire package like what is done for `System.IO.Compression.TestData` above.

Reply:
Yup, the documentation had me confused. I've pushed an update that should now exclude them.